src/Sentry/Protocol/Envelopes/EnvelopeItem.cs

```diff
@@ -277,23 +268,10 @@ internal static EnvelopeItem FromClientReport(ClientReport report)
             Stream stream,
             CancellationToken cancellationToken = default)
         {
-            var buffer = new List<byte>();
-
-            var prevByte = default(int);
-            await foreach (var curByte in stream.ReadAllBytesAsync(cancellationToken).ConfigureAwait(false))
-            {
-                // Break if found an unescaped newline
-                if (curByte == '\n' && prevByte != '\\')
```

Comment:
Note that the `prevByte` check was extraneous.  An escaped newline is `'\\' + 'n'`, not `'\\' + '\n'`.